### DISCUSSION TITLE
Checking the ashes

### DISCUSSION THREAD

**Discussion by Eno-Scott** [bleachbit/bleachbit#1494] [Q&A]
I've downloaded the last portable version of Bleachbit from your site and checked the checksum with certutil.exe on my system but the two ashes were different,it could be a problem?
Windows 10 
Bleachbit 4.4.2 portable

I've also checked the ash that I've get whit virus total and it says that is clear,so why the twos are different ?
(Please be kind with me because I'm not sure if this is the right place for asking that)

**Comment by az0** [comment]
What is the hash value that you get? Was the exact file name they're
checking? Which algorithm are you using?

**Reply by Eno-Scott** [reply]
I'm sorry,I've checked the .exe file instead of the .zip archive,the checksum is correct,thanks for helping me
